gammagl/models/unimp.py

```diff
@@ -0,0 +1,24 @@
+import tensorlayerx as tlx
+import tlx.nn as nn
+from gammagl.layers import MultiHead
+
+class Unimp(tlx.nn.Module):
```

Comment:
Please add ``rst`` doc here refer to this link: https://docs.qq.com/pdf/DUXRTTU9tUnB1WnFB.